src/configuration/builder.rs

```diff
@@ -1,5 +1,7 @@
+use dprint_core::configuration::{
+    resolve_global_config, ConfigKeyMap, ConfigKeyValue, GlobalConfiguration, NewLineKind,
+};
 use std::collections::HashMap;
-use dprint_core::configuration::{GlobalConfiguration, resolve_global_config, NewLineKind, ConfigKeyMap, ConfigKeyValue};
```

Comment:
@rivy sorry, would you be able to revert the cargo fmt commit? Maybe one day I will switch to using it, but not today :)

Reply:
Actually, if you want to do a separate PR that adds formatting feel free. It should just use this plugin:

https://github.com/dprint/dprint-plugin-rustfmt/releases/tag/0.4.0

...with an indent width of "2" and line width of 160.

Reply:
Will do... I used a separate commit for that possibility.

Reply:
`dprint` config updated and `dprint fmt` done in first two commits.

Reply:
Oh, just noted the "separate" commit request.
I'll do it that way instead.

Reply:
Reformat split out into #50.